index.js

```diff
@@ -56,13 +56,18 @@ function getDefaultConfig(mainTemplate, templatesPath) {
       const marked = require('marked');
       const ramljsonexpander = require('raml-jsonschema-expander');
       const renderer = new marked.Renderer();
+      
       renderer.table = function (thead, tbody) {
         // Render Bootstrap style tables
         return `<table class="table"><thead>${thead}</thead><tbody>${tbody}</tbody></table>`;
       };
 
       // Setup the Nunjucks environment with the markdown parser
       const env = nunjucks.configure(templatesPath, { autoescape: false });
+      
+      if(config.setupNunjucks)
```

Comment:
You're not passing the `config` parameter into the `processRamlObj` function so this won't work. It also absolutely does not follow the code style of the project and this won't lint.